app/src/main/res/layout/view_indonesia_new_tab_section.xml

```diff
@@ -16,5 +16,6 @@
 
 <com.duckduckgo.common.ui.view.MessageCta xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/newTabIndonesiaMessage"
+    android:layout_marginTop="@dimen/keyline_1"
```

Comment:
margins are now inside the view, otherwise they are always present in the New Tab page, even if this view is set to gone.